server/src/main/java/org/mixit/cesar/site/web/app/SessionWriterController.java

```diff
@@ -0,0 +1,131 @@
+package org.mixit.cesar.site.web.app;
+
+import static org.mixit.cesar.site.config.CesarCacheConfig.CACHE_LIGHTNINGTALK;
+import static org.mixit.cesar.site.config.CesarCacheConfig.CACHE_SPEAKER_LT;
+
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.mixit.cesar.security.model.Account;
+import org.mixit.cesar.security.service.authentification.CurrentUser;
+import org.mixit.cesar.security.service.autorisation.Authenticated;
+import org.mixit.cesar.security.service.exception.ForbiddenException;
+import org.mixit.cesar.site.model.member.Member;
+import org.mixit.cesar.site.model.session.Format;
+import org.mixit.cesar.site.model.session.LightningTalk;
+import org.mixit.cesar.site.model.session.Session;
+import org.mixit.cesar.site.model.session.SessionLanguage;
+import org.mixit.cesar.site.repository.MemberRepository;
+import org.mixit.cesar.site.repository.SessionRepository;
+import org.mixit.cesar.site.repository.VoteRepository;
+import org.mixit.cesar.site.service.EventService;
+import org.mixit.cesar.site.web.dto.SessionResource;
+import org.mixit.cesar.site.web.dto.VoteDto;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cache.CacheManager;
+import org.springframework.context.ApplicationContext;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * Controller used to return the parameters used in the CFP
+ */
+@RestController
+@RequestMapping("/app/session")
+@Transactional
+public class SessionWriterController {
+
+    @Autowired
+    private SessionRepository sessionRepository;
+
+    @Autowired
+    private MemberRepository memberRepository;
+
+    @Autowired
+    private VoteRepository voteRepository;
+
+    @Autowired
+    private ApplicationContext applicationContext;
+
+
+    @Autowired
+    CacheManager cacheManager;
+
+    @RequestMapping(value = "/mylightnings")
+    @Authenticated
+    public List<SessionResource> getMyLightningsInfo() {
+        Member currentUser = applicationContext.getBean(CurrentUser.class).getCredentials().get().getMember();
+        return sessionRepository
+                .findAllMyLightningTalks(EventService.getCurrent().getId(), currentUser.getId())
+                .stream()
+                .map(s -> SessionResource.convert(s))
+                .collect(Collectors.toList());
+    }
+
+    @RequestMapping(value = "/lightnings/votes")
+    @Authenticated
+    public List<VoteDto> getLightningsVotes() {
+        Member currentUser = applicationContext.getBean(CurrentUser.class).getCredentials().get().getMember();
+        return voteRepository.findByMember(currentUser)
```

Comment:
Do we mix admin CFP votes on proposals and public votes on sessions ? I don't feel it as a good idea ?